packages/vitest/src/runtime/setup.ts

```
@@ -167,7 +168,8 @@ export async function runSetupFiles(config: ResolvedConfig) {
   await Promise.all(
     files.map(async (file) => {
       getWorkerState().moduleCache.delete(file)
-      await import(file)
+      if (existsSync(file))
+        await import(file)
```

We shouldn't silently skip this